@xen-orchestra/proxy/src/app/mixins/backups/_getTmpDir.js

```diff
@@ -0,0 +1,9 @@
+import Disposable from 'promise-toolbox/Disposable'
+import fromCallback from 'promise-toolbox/fromCallback'
+import tmp from 'tmp'
+import { rmdir } from 'fs-extra'
+
+export const getTmpDir = async () => {
+  const tmpDir = await fromCallback(tmp.dir)
+  return new Disposable(tmpDir, () => rmdir(tmpDir))
```

Comment:
This is not the way to remove a directory created by `tmp.dir` but it's related to this PR, I'll fix this separately.

Reply:
Not possible to do it on an `async` way using `fromCallback`.